src/tasks/tasks.py

```diff
@@ -196,7 +196,7 @@ def __init__(self, name, tokenizer_name):
         self.name = name
         self._tokenizer_name = tokenizer_name
         self.scorers = []
-
+        self.is_diagnostic = False
```

Comment:
I'd prefer to rename this to what the property that we actually care about is. "Diagnostic" is a pretty loose term; does this specifically refer to tasks for which we don't train on ? Tasks that use another task's classifier?

Reply:
IDK how to leave the comment on the code diff, but shouldn't `main.py:533` use this flag instead of checking on the task class?

Reply:
Good point- changing in the code

Reply:
Agree with Alex's initial suggestion. Maybe use something like 'eval_only_task = 1' or 'can_be_trained = 0'.

Also, this'll likely be temporary—once we finish cleaning up the logic for diagnostic tasks that use classifiers from other tasks, we can simply test whether this task uses its own classifier, and skip training if not.